src/Core/Grid/Action/Row/Type/LinkRowAction.php

```diff
@@ -50,12 +50,18 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver
             ->setRequired([
                 'route',
```

Comment:
sounds like you may have a specific object (ColumnLink() ?) to store these parameters. You can take a look at what I've done with ColumnFilterAction in my last pull request :+1: